components/Footer/Component.jsx

```diff
@@ -34,7 +34,7 @@ export default function Footer({ site, navigation }) {
   }, [isClient]);
 
   return (
-    <footer className="su-shadow-[0px_-3px_6px_0px_rgba(0,0,0,0.1)] su-pt-70 lg:su-pt-[17.5rem] dark:su-shadow-[0px_-3px_6px_0px_rgba(46,45,41,0.5)]">
+    <footer className="su-shadow-[0px_-3px_6px_0px_rgba(0,0,0,0.1)] su-pt-70 lg:su-pt-[17.5rem] dark:su-shadow-[0px_-3px_6px_0px_rgba(46,45,41,0.5)] su-bg-white dark:su-bg-black-true">
```

Comment:
This is my only global Footer change which shouldn't make a difference on the front end. It's just to prevent it looking like this when using the Campaign page:
![Screenshot 2024-07-16 at 4 49 36 PM](https://github.com/user-attachments/assets/0a8002e9-e8aa-4573-b055-b9dcebb03c67)